src/peakWatcher.cpp

```diff
@@ -873,19 +874,10 @@ void saveExtensionConfig(ProjectStateContext* ctx, bool isUndo,
 
 void BeginLoadProjectState (bool isUndo, struct project_config_extension_t* reg){
 	//clean up configuration data for dead projects
-	vector<ReaProject*> openProjects;
-	for (int i = 0;; i++){
-		ReaProject* project = EnumProjects(i, nullptr, 0);
-		if (!project) {
-			break;
-		}
-		openProjects.push_back(project);
-	}
-	for (auto& [project, _]: watchers){
-		if (find(openProjects.begin(), openProjects.end(), project) == openProjects.end()){
-			watchers.erase(project);
-		}
-	}
+	erase_if(watchers, [](const auto& item) {
+auto const& [project, _] = item;
```

Comment:
Ug. I wish we could use std::ignore in decomposition/structured bindings. I can't decide whether this or `project = item->first` looks cleaner. I'm surprised the compiler doesn't complain about an unused variable here.

Reply:
This was left over from the loop, I think `project = item.first` is much better. BTW, it looks like `_` will be an official placeholder / wild card in the future. [A nice placeholder with no name](https://www.open-std.org/jtc1/sc22/wg21/docs/papers/2023/p2169r3.pdf) Compilers apparently don't complain about unused variables in structured binding because it is common to ignore one or more of them.